src/store/mutations.js

```diff
@@ -358,9 +363,8 @@ function SOCKET_ONOPEN(state, event) {
       });
 
       // Ensure trade history is up-to-date on reconnect. (may have dropped some trades during disconnect)
-      this.dispatch('fetchTradeHistory', {
-        marketName: state.currentMarket.marketName,
-        isRequestSilent: true,
+      this.dispatch('fetchMarkets', {
+        done: () => {},
```

Comment:
I left this random function here, only because I'm not sure what would go here.

Reply:
It needs to get new trade information it missed while websocket was disconnected.

Reply:
I handled the anonymous function.